test/jasmine/tests/hover_spikeline_test.js

```diff
@@ -34,6 +38,20 @@ describe('spikeline', function() {
             expect(d3.selectAll('circle.spikeline').size()).toEqual(1);
         });
 
+        it('draws lines and markers on enabled axes w/o tick labels', function(done) {
+            Plotly.relayout(gd, {
+                'xaxis.showticklabels': false,
+                'yaxis.showticklabels': false
+            })
+            .then(function() {
+                Fx.hover('graph', {xval: 2, yval: 3}, 'xy');
+                expect(d3.selectAll('line.spikeline').size()).toEqual(4);
+                expect(d3.selectAll('circle.spikeline').size()).toEqual(1);
```

Comment:
is it worth verifying that these things have reasonable dimensions/positions?

Reply:
Good call. done in 92b606b